### PR TITLE
HN 007 : Add static Auth

### PR DESCRIPTION
**Pull request by AsminBudha**
1. Add SignIn and SignUp button
2. Add storage in local storage.

### CODE REVIEW

File: src/AppRouter.js [hunk 1]
```diff
@@ -25,14 +26,15 @@ class AppRouter extends React.Component {
 
     return (
       <Router>
-        <div>
-          <Header />
+        <div className='container-fluid'>
+          <Route path='/' component={(props) => <Header {...props} />} />
```
Comment (badyakarprabin) [inline src/AppRouter.js:30]: Why not 
``` <Header/> ```

File: src/components/Header.js [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+  }
 
+  /**
+   * JSX for header.
+   *
+   * @returns
+   * @memberof Header
+   */
+  render() {
+
```
Comment (badyakarprabin) [inline src/components/Header.js]: Remove line

File: src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    const temp = users.filter((item) =>
+      item.email === emailVal && item.password === passwordVal);
+
+    if (temp.length > 0) {
+      localStorage.setLocalStorage(appConstants.IS_LOGGED_IN_KEY, true);
+      this.props.history.push('');
+    } else {
+      alert('Username or Password didn\'t match');
+    }
+
```
Comment (badyakarprabin) [inline src/components/Login.js]: Remove

File: src/components/NewsItem.js [hunk 4]
```diff
@@ -65,34 +66,45 @@ class NewsItem extends React.Component {
...
-            <Link to={
-              {
-                pathname: `${appConstants.API_ITEM}/${id}`,
-                state: { data: data }
-              }}
+        <div className='card card-wrapper'>
+          <div className='card-header'>{title}</div>
+          <div className='card-body'>
+            <h6 className='blockquote-footer'>
+              <cite title={by} >{by}</cite>
```
Comment (badyakarprabin) [inline src/components/NewsItem.js]: ``` {by}> ```

File: src/components/SinglePost.js [hunk 5]
```diff
@@ -12,9 +12,12 @@ const SinglePost = (props) => {
...
   return (
-    <div>
-      <h2>{title}</h2>
-      <Comments data={kids} />
+    <div className='card'>
+      <div className='card-header'>{title}</div>
+      <div className='card-body'>
+        <Comments data={kids} />
+      </div>
+
```
Comment (badyakarprabin) [inline src/components/SinglePost.js]: Remove

File: src/index.css [hunk 6]
```diff
@@ -21,5 +36,36 @@ ul{
...
+  margin: 5px auto;
+}
+
+.card-wrapper:nth-child(1){
+  margin-top: 0px;
+}
+
+.btn-signup{
+  margin-left: 15px;
+}
```
Comment (badyakarprabin) [inline src/index.css]: Next line

**Review by badyakarprabin — CHANGES_REQUESTED** [review]

File: src/AppRouter.js (hunk 1 above)
Comment (AsminBudha) [reply src/AppRouter.js:30]: Need routes path so that when click on sign out button inside header which routes to root url.

**Pull request merged**